cmake/traccc-ctest.sh.in

```diff
@@ -0,0 +1,17 @@
+#!@BASH_EXECUTABLE@
+#
+# TRACCC library, part of the ACTS project (R&D line)
+#
+# (c) 2024 CERN for the benefit of the ACTS project
+#
+# Mozilla Public License Version 2.0
+
+# Propagate errors.
+set -e
+set -o pipefail
+
+# Run every command through the time command. Recording the time, memory, etc.
+# used by each and every build command.
+@TIME_EXECUTABLE@ --verbose \
+   --output=@CMAKE_CURRENT_BINARY_DIR@/traccc_build_performance.log --append \
+   @CMAKE_CTEST_COMMAND@ $*
```

Comment:
Does this ensure that the time executable is used rather than the builtin? Recommend the following instead:

```suggestion
command @TIME_EXECUTABLE@ --verbose \
   --output=@CMAKE_CURRENT_BINARY_DIR@/traccc_build_performance.log --append \
   @CMAKE_CTEST_COMMAND@ $*
```

Reply:
It does ensure that as-is. `TIME_EXECUTABLE` comes from a [find_program](https://cmake.org/cmake/help/latest/command/find_program.html) call. Which cannot be used to look for built-in commands. And it returns the absolute path to the binary, like `/usr/bin/time`.

This is exactly why I'm leaving it to CMake to find the executable in that way... 🤔

Reply:
The CMake documentation doesn't explicitly say it always returns the absolute path, so I don't trust this. I'd add the `command` to the shell script to be sure.

Reply:
Sure. Not the hill I want to die on. 🤔